spec/models/ingest/orchestrator_spec.rb

```diff
@@ -58,8 +58,35 @@
 
         orchestrator = Ingest::Orchestrator.new(file)
         orchestrator.perform
+      end
+
+      it 'has our "other" rows plus some invoices and orders' do
+        aggregate_failures do
+          expect(submission.entries.where(entry_type: 'invoice').count).to eql(1)
+          expect(submission.entries.where(entry_type: 'order').count).to eql(3)
+          expect(submission.entries.where(entry_type: 'other').count).to eql(5)
+
+          expect(file.rows).to eql(1 + 3 + 5)
+        end
+      end
 
-        expect(submission.entries.where(entry_type: 'other').count).to eql(5)
+      context 'without the Orders' do
```

Comment:
Should say "without the `OrderFields` block, as that's what 3774.2 is really doing. Maybe we could also rename that.